package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "@avenga/eslint-config",
+  "version": "1.0.0",
+  "description": "node typescript linting configs from avenga",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/avenga/linting-configs.git"
+  },
+  "keywords": [
+    "avenga",
+    "typescript",
+    "ts",
+    "config",
+    "lint",
+    "linting",
+    "linter",
+    "backend",
+    "frontend"
+  ],
+  "author": "Avenga",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/avenga/eslint-config/issues"
+  },
+  "homepage": "https://github.com/avenga/eslint-config#readme",
+  "peerDependencies": {
+    "@typescript-eslint/eslint-plugin": "~2.28.0",
+    "@typescript-eslint/parser": "~2.28.0",
+    "eslint": ">=6.8"
+  },
+  "dependencies": {},
+  "devDependencies": {}
```

Comment:
Do we need this empty objects here?

Reply:
we do not need them as long as they are empty objects
should they be empty?